src/proto/proto_common.cpp

```diff
@@ -896,9 +896,15 @@ void print_help(std::ostream& os)
        "  --ltfb_verbose \n"
        "      Increases number of per-trainer messages that are reported\n"
        "  --ckpt_dir=<string>\n"
-       "      Save to or reload from a specific checkpoint directory.\n"
+       "      Save to or restart from a specific checkpoint directory.\n"
        "      Additionally, sets the output directory for dumping weights.\n"
        "      Modifies callbacks: checkpoint, save_model, dump_weights\n"
+       "  --restart_dir=<string>\n"
+       "      Restart from a checkpoint found in the given directory.\n"
+       "      If the directory doesn't exist or doesn't contain a checkpoint,\n"
+       "      an error will be thrown.\n"
+       "  --restart_dir_is_fullpath=<bool>\n"
```

Comment:
Add this to `allowed_flags` in `tools.get_command`.

Reply:
Fixed.